aptos-move/move-examples/mint_nft/step1/sources/nft_step1.move

```diff
@@ -0,0 +1,144 @@
+/// This module is the step 1 of our NFT Move tutorial. In this module, we go over how to create a collection, a token,
+/// and mint a token to a receiver.
+///
+/// How to interact with this module:
+/// 1. Create a source account.
```

Comment:
nit: I think we should just call this "Create an account". source account is a new term and can be confusing. cc @clay-aptos